places/pismo-beach-ca.md

```diff
@@ -0,0 +1,9 @@
+## Pismo Beach, CA
```

Comment:
Ventana Grill


From: TDKurt [mailto:notifications@github.com]
Sent: Tuesday, March 22, 2016 9:57 AM
To: githubteacher/developers-open-march16
Cc: Abernathy, Linda
Subject: Re: [developers-open-march16] Lindaab hometown Pismo Beach, CA (#65)


In places/pismo-beach-ca.md<https://github.com/githubteacher/developers-open-march16/pull/65#discussion_r57024646>:

> @@ -0,0 +1,9 @@

> +## Pismo Beach, CA

@lindaab<https://github.com/lindaab> Where is a good place to eat?

—
You are receiving this because you were mentioned.
Reply to this email directly or view it on GitHub<https://github.com/githubteacher/developers-open-march16/pull/65/files/793a4d216c4eaccc53de728a0c1c2ce316133947#r57024646>